labkey-client-api/src/org/labkey/remoteapi/domain/ConditionalFormat.java

```diff
@@ -0,0 +1,110 @@
+package org.labkey.remoteapi.domain;
+
+import org.json.simple.JSONObject;
+
+public class ConditionalFormat
+{
+    private String queryFilter;
+    private String textColor = "";
+    private String backgroundColor = "";
+    private Boolean bold = false;
+    private Boolean italic = false;
+    private Boolean strikethrough = false;
+
+    public ConditionalFormat(String queryFilter, String textColor, String backgroundColor, Boolean bold, Boolean italic, Boolean strikethrough)
+    {
+        this.queryFilter = queryFilter;
+        this.textColor = textColor;
+        this.backgroundColor = backgroundColor;
+        this.bold = bold;
+        this.italic = italic;
```

Comment:
For the Boolean variables, you might need to null check otherwise you may be forced to null check on serialization.